CHANGELOG.rst

```diff
@@ -4,6 +4,18 @@ Changelog
 in development
 --------------
 
+Changed
+~~~~~~~
+
+* When creating a pack environment during the pack installation, we now pass ``--no-download`` flag
+  to the ``virtualenv`` binary. This way version of pip, wheel and distutils which is already
+  available on the system is used instead of downloading the latest stable versions from PyPi.
```

Comment:
A small change to this would be saying:
> This way version of pip, wheel and distutils which is enforced by virtualenv version is used

Since virtualenv installs "pinned" version he associates with, according to https://virtualenv.pypa.io/en/stable/changes/